src/Runner.Worker/JobRunner.cs

```diff
@@ -50,10 +50,12 @@ public async Task<TaskResult> RunAsync(AgentJobRequestMessage message, Cancellat
             if (message.Variables.TryGetValue(Constants.Variables.System.OrchestrationId, out VariableValue orchestrationId) &&
                 !string.IsNullOrEmpty(orchestrationId.Value))
             {
-                if (!HostContext.UserAgents.Any(x => string.Equals(x.Product.Name, "OrchestrationId", StringComparison.OrdinalIgnoreCase)))
+                HostContext.LoadDefaultUserAgents();
```

Comment:
we don't need this, `LoadDefaultUserAgents` is called when we create the HostContext.